packages/core/test/unit/html/NodeProcessor.data.ts

```diff
@@ -92,18 +69,38 @@ export const PROCESS_QUESTION_ATTRIBUTES_EXPECTED = `
 </question>
 `;
 
-export const PROCESS_QUESTION_ATTRIBUTES_NO_OVERRIDE = `
-<question header="**header**" hint="**hint**" answer="**answer**">
+export const PROCESS_QUESTION_HEADER_SLOT_TAKES_PRIORITY = `
+<question header="**header**">
```

Comment:
NIT: Since this text is overridden - maybe you can change the overridden text to "lorum ipsum" or something like in the other tests? This was confusing to me because I didn't realise the difference between what was overridden for a while.

Similar for hint & answer below